web-ui/src/components/volunteer/VolunteerTables.css

```diff
@@ -30,31 +47,46 @@
   td,
   th {
     padding: 0.5rem;
+    color: black; /* Ensures text color remains black */
   }
 
   th {
     background-color: var(--oci-orange);
     border-right: 1px solid var(--oci-light-blue);
   }
+
   th:last-child {
     border-right: none;
   }
-  .actions-th {
-    width: 6rem;
-  }
 
   tr:nth-child(even) {
     background-color: #eee;
   }
+
   tr:nth-child(odd) {
     background-color: #ffffff;
   }
 
+  .actions-th {
+    width: 6rem;
+  }
+
   .warning {
     font-style: italic;
   }
 }
 
+
+.volunteer-tables .MuiIconButton-root {
+  color: black;
```

Comment:
Same here...should be: `var(--checkins-palette-text-primary)`

Reply:
The problem with `var(--checkins-palette-text-primary)` in this case is that the table background is white in both light and dark mode.  And, in dark mode, this changes the text color to white.  Should the table background be something other than white when in dark mode?

Reply:
Is it not being altered appropriately by the theming?

Reply:
@thelenw Ok, I looked at this and yes...the table background is definitely not being altered appropriately. I would see what variables are being used for the other backgrounds and see if there are appropriate values available. My guess is that there is one for the primary background, but maybe not one for the alternating rows. You may need to find a color that works for that and define a new alternate row background variable for use by both themes.

Reply:
@thelenw I took a look at this, and I think I have a fix for you. I'll push shortly.